client/my-sites/site-settings/analytics/form-google-analytics-jetpack.jsx

```diff
@@ -53,7 +53,7 @@ const GoogleAnalyticsJetpackForm = ( {
 	const analyticsSupportUrl = 'https://jetpack.com/support/google-analytics/';
 	const nudgeTitle = translate( 'Connect your site to Google Analytics' );
 	const wooCommercePlugin = find( sitePlugins, { slug: 'woocommerce' } );
-	const wooCommerceActive = wooCommercePlugin ? wooCommercePlugin.active : false;
+	const wooCommerceActive = wooCommercePlugin ? wooCommercePlugin.sites[ siteId ].active : false;
```

Comment:
That would work, but it would require changing how `sitePlugins` is done in `client/my-sites/site-settings/analytics/form-google-analytics.jsx`. I think that could just be replaced with a connected `wooCommerceActive` prop, but for this PR I want to keep the scope to just translating existing code to use the new TypeScript selectors. I can make an issue for this change if that seems worth doing.

Reply:
Sounds good. A TODO comment in the PR could also work if an issue feels too much.